Configuration/PyReleaseValidation/python/relval_2017.py

```diff
@@ -15,8 +15,8 @@
 
 numWFStart=10000
 numWFSkip=200
-#2017 WFs to run in IB (TenMuE_0_200, TTbar, ZEE, MinBias, TTbar PU, ZEE PU)
-numWFIB = [10021.0,10024.0,10025.0,10026.0,10023.0,10224.0,10225.0]
+#2017 WFs to run in IB (TenMuE_0_200, TTbar, ZEE, MinBias, TTbar PU, ZEE PU, TTbar design)
+numWFIB = [10021.0,10024.0,10025.0,10026.0,10023.0,10224.0,10225.0,10424.0]
```

Comment:
my guess is tht https://github.com/makortel/cmssw/blob/77eecdbe8a344f294ced034a29773c8d8a6d0d27/Configuration/PyReleaseValidation/python/relval_steps.py#L1876 is where it is setting invalid dataset for TTbar_13TeV_TuneCUETP8M1_2017Design_GenSimFull